### PR TITLE
Update theme and add sign up config

### PR DESCRIPTION
**Pull request by caylahamann**
Please follow [conventional commit standards](https://www.conventionalcommits.org/en/v1.0.0/)
in your commit messages and pull request title.

## Give us some context

* What problems does this PR solve?
* Add any context that will help us review your changes such as testing notes,
  links to related docs, screenshots, etc.
* If your issue relates to an existing GitHub issue, please link to it.

### CODE REVIEW

File: gatsby-config.js [hunk 1]
```diff
@@ -161,6 +161,15 @@ module.exports = {
             }),
           },
         },
+        signup: {
+          environment: process.env.ENVIRONMENT || 'staging',
```
Comment (roadlittledawn) [inline gatsby-config.js:165]: we need to add this env var to gatsby cloud. should it be `production`?

Comment (roadlittledawn) [reply gatsby-config.js:165]: looks like it should be `production`. but please double check me

Comment (caylahamann) [reply gatsby-config.js:165]: yes it should be production, but don't we already have this set in the site??

Comment (caylahamann) [reply gatsby-config.js:165]: i just added another variable

Comment (roadlittledawn) [reply gatsby-config.js:165]: ideally we'd operate from one env var to indicate waht env we're in.

not 100% what would happen if window._nr_signup.environment is set to `development` though (which is the only other value we use for `GATSBY_NEWRELIC_ENV`.

in any case, we can proceed and dig into that further as a potential refactor if needed

**Review by roadlittledawn — APPROVED** [review]

**Pull request merged**